lib/socialcast/command_line/provision_user.rb

```diff
@@ -55,13 +56,32 @@ def provision
               request_params[:add_only] = 'true' if (@ldap_config.fetch('options', {})['add_only'] || @options[:add_only])
               resource.post request_params, :accept => :json
             end
-          rescue RestClient::Unauthorized => e
-            raise ProvisionError.new "Authenticated user either does not have administration privileges or the community is not configured to allow provisioning. Please contact Socialcast support to if you need help." if e.http_code == 401
+          rescue RestClient::Unauthorized, RestClient::Forbidden => e
+            raise ProvisionError.new provision_error_message(e.http_code)
           end
           log "Finished"
         end
         File.delete(output_file) if (@ldap_config.fetch('options', {})['delete_users_file'] || @options[:delete_users_file])
       end
+
+      private
+
+      def provision_error_message(error_code)
+        case error_code
+        when 401
+          <<-EOS.strip_heredoc
+            Received an "Unauthorized" error from the Socialcast server. Please check the following:
+            * Community has basic authentication enabled
+            * User has administration privileges
+            * User or External System is active
+            * Credentials and community domain are correct in #{Socialcast::CommandLine.credentials_file}
+          EOS
+        when 403
+          <<-EOS.strip_heredoc
+            Received a "Forbidden" error from the Socialcast server. Please check that your community has directory integration enabled.
+          EOS
```

Comment:
@larsklevan I can add that, but since we're only rescuing `RestClient::Unauthorized` and `RestClient::Forbidden`, I can't think of a way that could happen.